codegen/src/main/java/software/amazon/awssdk/codegen/poet/model/ExceptionProperties.java

```diff
@@ -44,7 +47,9 @@ public static List<MethodSpec> builderImplMethods(ClassName className) {
                 builderImplMethods(className, "requestId", String.class),
                 builderImplMethods(className, "statusCode", int.class),
                 builderImplMethods(className, "cause", Throwable.class),
-                builderImplMethods(className, "writableStackTrace", Boolean.class));
+                builderImplMethods(className, "writableStackTrace", Boolean.class),
+                builderImplMethods(className, "numAttempts", Integer.class),
+                builderImplGetterMethods("numAttempts", Integer.class));
```

Comment:
Looks like we don't have builderGetterMethods for other properties above, is there any reason?